primitives/src/runtime_api.rs

```diff
@@ -210,6 +210,9 @@ sp_api::decl_runtime_apis! {
 			-> Option<ppp::ValidationCodeHash>;
 
 
+		// FIXME: Docs; move to v3
+		fn session_ee_params_by_parent_hash(parent_hash: pcp::v2::Hash) -> Option<vstaging::ExecutorParams>;
```

Comment:
If you want this in v3 just add `#[api_version(3)] here. Then the method won't be available in v2.